### PR TITLE
Converge external and internal deployment of firebird in one

### PR DESCRIPTION
**Pull request by DeLaGuardo**
healthcheck and metrics will be exposed only on 4001 port for internal use

fix xapix-io/firebird-next#351

### CODE REVIEW

File: xapix/charts/firebird/templates/firebird-deployment.yaml [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
-  name: "{{ .Release.Name }}-firebird-external"
+  name: "{{ .Release.Name }}-firebird"
   labels:
     heritage: {{ .Release.Service | quote }}
     release: {{ .Release.Name | quote }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     tier: public
     app: {{ .Chart.Name | quote }}
+  annotations:
+    prometheus.io/scrape: "true"
```
Comment (mreinsch) [inline xapix/charts/firebird/templates/firebird-deployment.yaml]: I don't think we need those anymore? At least I think I removed them from all other deployments a while ago.

File: xapix/charts/firebird/templates/service-monitor.yaml [hunk 2]
```diff
@@ -1,18 +1,6 @@
...
-    matchLabels:
-      release: {{ .Release.Name | quote }}
-      component: firebird-external
-  endpoints:
-  - port: http
----
-apiVersion: monitoring.coreos.com/v1
-kind: ServiceMonitor
 metadata:
   name: "{{ .Release.Name }}-firebird-internal"
```
Comment (mreinsch) [inline xapix/charts/firebird/templates/service-monitor.yaml]: I think we can also just call this `"{{ .Release.Name }}-firebird"` ?

Comment (DeLaGuardo) [reply xapix/charts/firebird/templates/service-monitor.yaml]: sure

File: xapix/charts/firebird/templates/firebird-deployment.yaml (hunk 1 above)
Comment (DeLaGuardo) [reply xapix/charts/firebird/templates/firebird-deployment.yaml]: deleted

**Pull request merged**